src/jdk.jdwp.agent/share/native/libjdwp/debugInit.c

```diff
@@ -74,6 +80,9 @@ static jboolean initOnUncaught = JNI_FALSE; /* init when uncaught exc thrown */
 
 static char *launchOnInit = NULL;           /* launch this app during init */
 static jboolean suspendOnInit = JNI_TRUE;   /* suspend all app threads after init */
+#if defined(J9VM_OPT_CRIU_SUPPORT)
+static jboolean suspendOnRestore = JNI_TRUE;   /* suspend all app threads after VM restored */
```

Comment:
Changed default for `suspendOnRestore` to be true.

Reply:
Please collapse the repeated spaces:
```c
static jboolean suspendOnRestore = JNI_TRUE; /* suspend all app threads after VM restored */
```